workflows/iuf/operations/management-nodes-rollout/management-storage-nodes-rollout.yaml

```diff
@@ -64,7 +64,7 @@ spec:
             templateRef:
               name: workflow-template-record-time-template
               template: record-time-template
-          - name: verify-storage-images-and-configuration
+          - name: INFO-to-read
```

Comment:
I think this is a good idea generally, however, INFO-to-read is unique for each stage. It provides information about what that stage is doing. It would add more complication to try to have 1 common template with different output based on node type. In my opinion, it should be left as is.

Reply:
I misread your comment, yes the general template should be removed. I will remove it.